community/fingerprinters/web/scripts/updater/zabbix/update.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
Hi @hh-hunter, instead of copying the entire fingerprinting plugin to the `/community` folder, can you please make a PR that contributes to the existing web fingerprinting plugin, i.e. submit only the `zabbix` automation script and the generated `.binproto` file? Thanks!

Reply:
Can I understand it this way? The `community/fingerprinters/web/scripts/updater/update.sh`、`community/fingerprinters/web/scripts/updater/zabbix/` and the `community/ fingerprinters/web/src/main/resources/fingerprinters/web/ data/zabbix.binproto` is submitted to the new pr request. So what is the destination path for the commit? Is there a naming convention? And another question I mentioned before, I submitted application #233 before #244, both applications are for the same vulnerability, but you agreed to #244 , but my application #233 is on hold, please tell me what's going on here?